src/tests/specs/iteration.spec.ts

```diff
@@ -9,20 +9,17 @@ describe('Iteration test', () => {
 
   beforeEach( async () => {
     await support.desktopTestSetup();
-    let token = encodeURIComponent(JSON.stringify({
-      access_token: "somerandomtoken",
-      expires_in: 1800,
-      refresh_token: "somerandomtoken",
-      token_type: "bearer"
-    }));
-    let planner_url = browser.baseUrl + "/?token_json=" + token;
-    planner = new PlannerPage(planner_url);
+    planner = new PlannerPage(browser.baseUrl);
     await planner.openInBrowser();
+    // This is necessary since the planner takes time to load on prod/prod-preview
+    await browser.sleep(12000);
+    await planner.ready();
   });
 
   it('should create a new iteration', async () => {
     await planner.sidePanel.createNewIteration();
-    await planner.iteration.addNewIteration(c.newIteration,c.iteration1);
+    await planner.iteration.addNewIteration(c.newIteration,c.iteration3);
+    await planner.iteration.createIterationButton.untilHidden();
```

Comment:
Can we move this line to `addNewIteration` method? This will help in keeping the tests clear of all unnecessary waits

Reply:
done!